tests/unit/src/main/java/org/mule/tck/util/MuleContextUtils.java

```diff
@@ -171,7 +172,7 @@ public static MuleContextWithRegistry mockMuleContext() {
 
     StreamingManager streamingManager = mock(StreamingManager.class, RETURNS_DEEP_STUBS);
     try {
-      MuleRegistry registry = mock(MuleRegistry.class, withSettings().lenient());
+      MuleRegistry registry = mock(MuleRegistryHelper.class, withSettings().lenient());
```

Comment:
doesn't this affect any other tests?

Reply:
Runned all the tests and found no side effects (all passing) Will check again just in case.

Reply:
All tests passing, no side effects.